doc/development_process/gh_labels.rst

```diff
@@ -0,0 +1,128 @@
+Minimum PR review time
+**********************
+
+=============  ===============================================================
+Labels         ``Hot Fix``, ``Trivial PR``, ``Maintainer``,
+               ``Security Review``, ``TSC``
+Applicable to  PRs only
+Description    Depending on the PR complexity, an indication of how long a merge
+               should be held to ensure proper review. See
+               :ref:`review process <review_time>`
+=============  ===============================================================
+
+Issue priority labels
+*********************
+
+=============  ===============================================================
+Labels         ``priority:{high|medium|low}``
+Applicable to  Issues only
+Description    To classify the priority/importance of a bug or feature
+=============  ===============================================================
+
+Note: Issue priorities are decided/changed during the bug-triage or TSC
+meetings.
+
+Miscellaneous labels
+********************
+
+For both PRs and issues
+=======================
+
++-------------------+----------------------------------------------------------+
+|``Bug``            | The issue is a bug, or the PR is fixing a bug            |
++-------------------+----------------------------------------------------------+
+|``Coverity``       |A Coverity detected issue or its fix                      |
++-------------------+----------------------------------------------------------+
+|Waiting for        | The Zephyr developers are waiting for the submitter to   |
+|author response    | respond to a question, or address an issue.              |
++-------------------+----------------------------------------------------------+
+|``RFC``            |The author would like input from the community. For a PR  |
+|                   |it should be considered a draft                           |
++-------------------+----------------------------------------------------------+
+|``LTS``            |Long term release branch related                          |
++-------------------+----------------------------------------------------------+
+|``In progress``    |For PRs: is work in progress and should not be merged yet.|
+|                   |For issues: Is being worked on                            |
++-------------------+----------------------------------------------------------+
+|``EXT``            | Related to an external component (in ``ext/``)           |
++-------------------+----------------------------------------------------------+
+
+PR only labels
+==============
+
+================ ===============================================================
+``DNM``          This PR should not be merged (Do Not Merge).
+                 For work in progress, GitHub "draft" PRs are preferred
+``Stale PR``     PR which seems abandoned, and requires attention by the author
+``Blocked``      Blocked by another PR or issue
+``needs review`` The PR needs attention from the maintainers
```

Comment:
Is this an opportunity to have all labels with a consistent upper-case first letter?

Reply:
I can certainly fix that :)